src/components/GraphicsMenu.tsx

```diff
@@ -180,14 +221,30 @@ export default class GraphicsMenu extends React.Component<
     if (this.state.invalidName) {
       validInputColor = "danger";
     }
+
+    let selectedVariableName = "";
+    let selectedVariable: any = undefined;
+    let axisNames = new Array<string>();
+    if(this.props.varTracker.selectedVariables.length > 0){
+      selectedVariableName = this.props.varTracker.selectedVariables[0].slice(0, this.props.varTracker.selectedVariables[0].length/2);
```

Comment:
I have no idea why I have to do this, but if the selected variable is "hus," then the variable tracker things that its "hushus."

Reply:
Variables consist of an alias, which can be changed by the user and is what they see, then they also have a 'name' which is how they are found within a file. I had to create the alias so that users could rename their variable without breaking their ability to reload the variable from a file. If you want to display only the user selected name (probably what you want), try using: selectedVariableName = this.props.varTracker.selectedVariables[0].alias

Reply:
Variables consist of an alias, which can be changed by the user and is what they see, then they also have a 'name' which is how they are found within a file. I had to create the alias so that users could rename their variable without breaking their ability to reload the variable from a file. If you want to display only the user selected name (probably what you want), try using: selectedVariableName = this.props.varTracker.selectedVariables[0].alias